docs/python/tutorial-azure-functions.md

```diff
@@ -0,0 +1,348 @@
+    - Select **Anonymous** for the authorization level, which makes the function publicly accessible to anyone.
+
+1. After a short time you see a message that the new project was created. In the **Explorer** you see the subfolder created for the function, and VS Code opens the `__init__.py` file that contains the default function code:
+
+    ![Result of creating a new Python functions project](images/functions/project-create-results.png)
+
+1. When `__init__.py` opens, VS Code might tell you that you don't have a Python interpreter selected. In that case, use the **Python: Select Interpreter** command from the Command Palette and select the virtual environment in the local `.env` folder (which was created as part of the project).
+
+    ![Selecting the virtual environment created with the project](images/functions/select-venv-interpreter.png)
+
+> **Tip**: Whenever you want to create another function in the same project, use the **Create Function** command in the **Azure: Functions** explorer, or use the **Azure Functions: Create Function** command from the Command Palette. Both commands prompt you for a function name (which is the name of the endpoint), then creates a subfolder with the default files.
+>
+> ![New Function command in the Azure: Functions explorer](images/functions/function-create-new.png)
+
+## Examine the code files
+
+In the newly-created function subfolder you see three files: `__init__.py`, which contains the function's code, `functions.json`, which describes the function to Azure Functions, and `sample.dat`, which demonstrates that a function subfolder can contain additional files if needs be. You won't be using `sample.dat`, so you can delete it if you want.
+
+Let's look at `functions.json` first, then the code in `__init__.py`.
+
+### functions.json
+
+The functions.json file provides the necessary configuration information for the function itself:
+
+```json
+{
+  "scriptFile": "__init__.py",
+  "bindings": [
+    {
+      "authLevel": "anonymous",
+      "type": "httpTrigger",
+      "direction": "in",
+      "name": "req",
+      "methods": [
+        "get",
+        "post"
+      ]
+    },
+    {
+      "type": "http",
+      "direction": "out",
+      "name": "$return"
+    }
+  ]
+}
+```
+
+You can see that `scriptFile` identifies the startup file for the code, which must contain an function named `main`. You can factor your code into multiple files so long as the file specified here contains a `main` function.
+
+The `bindings` element the contains two objects, one to describe incoming requests, and the other to describe the HTTP response. You can see that for incoming requests (`"direction": "in"`), the function responds to HTTP get or post requests and doesn't require authentication. The response (`"direction": "out"`) is an HTTP response that returns whatever value is returned from the `main` function.
```

Comment:
COPY (passive): Remove `You can see that`
COPY (style): Use `GET` and `POST` when referring to HTTP methods (would recommend stylizing inside `` ` ``)